pom.xml

```diff
@@ -23,6 +23,12 @@
 		<log4j.version>2.17.1</log4j.version>
 		<bson.version>2.3</bson.version>
 		<fusionauth.version>5.3.0</fusionauth.version>
+		<jcommander>1.82</jcommander>
```

Comment:
it should be <jcommander.version>, refer to line number 25, do the same with others on line 27, 28 and so on.

Reply:
done